doozerlib/cli/images_streams.py

```diff
@@ -124,6 +124,13 @@ def images_streams_mirror(runtime, streams, only_if_missing, live_test_mode, dry
                     print(f'For {upstream_entry_name}, would have run: {arm_cmd}')
                 else:
                     exectools.cmd_assert(arm_cmd, retries=3, realtime=True, timeout=1800)
+            # mirror builder image streams to priv namespace
+            if config.upstream_image_mirror is not Missing:
```

Comment:
The ticket requests this to be a list of destinations. It will be useful to be able to mirror to multiple destinations if we need to.

Reply:
updated, treat it as a list in https://github.com/openshift-eng/ocp-build-data/pull/3347/files